src/sass/modules/_m-form-elements.scss

```diff
@@ -75,9 +75,9 @@ button.form-button-disabled {
   // See 18F/web-design-standards#1490
   // This fixes things in the meantime.
   [type="checkbox"] {
-    height: auto !important;
+    height: 1.8rem !important;
```

Comment:
@raquelromano-gov No, I'm not sure, but I don't _think_ it will simply because it now matches the `input[type="checkbox"] + label::before`'s size, and that's the bit that visually looks like the checkbox. The `::before` size comes from the imported [USWDS style](https://github.com/18F/web-design-standards/blob/staging/src/stylesheets/elements/_inputs.scss#L165).

I'll see if I can't follow the instructions @ayaleloehr linked to and test out the messages.